test/microsoft_test.go

```diff
@@ -20,6 +21,7 @@ func TestMicrosoft(t *testing.T) {
 			&helm.Options{
 				SetStrValues: map[string]string{
 					"pachd.storage.backend":             "MICROSOFT",
+					"etcd.storage.backend":              "MICROSOFT",
```

Comment:
Is `etcd.storage.backend` referred to anywhere? I see that the `etcd` manifests refer to `pachd.storage.backend`.

@chainlink, what do you think: should storage be configurable separately for `pachd` and `etcd` (more flexible) or should it be global (simpler)?

Reply:
great point. IMHO it should probably be global

Reply:
Agree with global, we can break them out when there is a request for it.

It does raise an interesting question though. Right now we're conflating object store with cloud provider (which addresses almost all cases, I've never heard of a AWS cluster hitting a GCP Bucket for example) I'm not sure we want to continue to do that, or if we do, we should make it more explicit (by moving it out of `pachd` perhaps?)